config/db.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect('mongodb://localhost:27017/noteApp');
```

Comment:
You can also define this as an environment variable just to get used to the concept of environment variables.

Reply:
Done